0800-appendix-57793-vallee.Rmd

```diff
@@ -77,42 +116,71 @@ fpr::fpr_table_cv_summary_memo()
-
-
-## Structure Remediation and Cost Estimate {-}
-
-Should restoration/maintenance activities proceed, replacement of PSCIS crossing `r as.character(my_site)` with a bridge (`r fpr::fpr_my_pscis_info(col_pull = recommended_diameter_or_span_meters)`m span) is recommended. The cost of the work is estimated at \$`r format(fpr::fpr_my_cost_estimate(), big.mark = ',')` for a cost benefit of `r fpr::fpr_my_cost_estimate(col_pull = cost_net)` linear m/\$1000 and `r fpr::fpr_my_cost_estimate(col_pull = cost_area_net)` m^2^/\$1000.
-
+`r fpr_my_habitat_paragraph(loc = 'us')` The data for the upstream
+survey was misplaced and stream characteristics and measurements were
+obtained by reviewing photos. The habitat quality was excellent. There
+were abundant undercut banks and healthy mixed riparian vegetation
+throughout. The stream was wide and contained some pockets of gravel
+suitable for spawning resident species. The habitat was rated as
+`r fpr::fpr_my_priority_info(loc = 'us')` value as an important
+migration corridor containing suitable spawning habitat and having
+moderate rearing potential.
+
+## Structure Remediation and Cost Estimate {.unnumbered}
+
+Should restoration/maintenance activities proceed, replacement of PSCIS
+crossing `r as.character(my_site)` with a bridge
+(`r fpr::fpr_my_pscis_info(col_pull = recommended_diameter_or_span_meters)`m
+span) is recommended. The cost of the work is estimated at
+\$`r format(fpr::fpr_my_cost_estimate(), big.mark = ',')` for a cost
+benefit of `r fpr::fpr_my_cost_estimate(col_pull = cost_net)` linear
+m/\$1000 and `r fpr::fpr_my_cost_estimate(col_pull = cost_area_net)`
+m^2^/\$1000.
 
 <br>
 
-
-## Conclusion {-}
-
-There was `r fpr::fpr_my_priority_info(col_pull= upstream_habitat_length_km)`km of habitat modelled upstream of crossing `r as.character(my_site)` with areas surveyed rated as `r fpr::fpr_my_priority_info(sit = my_site, loc = 'us', col_pull = hab_value)` value for salmonid rearing and spawning. Crossing `r as.character(my_site)` was ranked as a `r fpr::fpr_my_priority_info(col_pull = priority)` priority for proceeding to design for replacement. 
-
+## Conclusion {.unnumbered}
+
+There was
+`r fpr::fpr_my_priority_info(col_pull= upstream_habitat_length_km)`km of
+habitat modelled upstream of crossing `r as.character(my_site)` with
+areas surveyed rated as
+`r fpr::fpr_my_priority_info(sit = my_site, loc = 'us', col_pull = hab_value)`
+value for salmonid rearing and spawning. Crossing
+`r as.character(my_site)` was ranked as a
```

Comment:
outlet drop doesn't change with flow level!

How about

As the culvert outlet drop was small, and because the relatively low elevation of the watershed may result in low flows in typical years ([see assessment photos from @gollner2013 here](https://a100.gov.bc.ca/pub/pscismap/imageViewer.do?assessmentId=57833)), `r as.character(my_site)` was ranked as a
`r fpr::fpr_my_priority_info(col_pull = priority)` priority for proceeding to design for replacement. Electrofishing upstream and downstream of the culvert is recommended to provide insight into fish community
composition and density which may justify a higher priority for site remediation.